src/lpython/parser/parser.yy

```diff
@@ -243,33 +244,21 @@ units
     ;
 
 script_unit
-    : statement sep   { $$ = SCRIPT_UNIT_STMT($1); }
-    | expr sep        { $$ = SCRIPT_UNIT_EXPR($1); }
-    ;
-/*
-statements
-    : single_stmt_list TK_NEWLINE { }
-    | sep TK_INDENT statements1 TK_DEDENT { }
+    : statement END_OF_FILE
```

Comment:
I added END_OF_FILE to resolve the following error:
```
syntax error: End of file is unexpected here
  --> examples/expr2.py:11:1
   |
11 | 
   | ^ 
```
This is not the correct approach to fix this, we need to figure out another way.